src/components/textarea/textarea.spec.tsx

```diff
@@ -23,18 +23,24 @@ import StyledTextarea from "./textarea.style";
 
 jest.mock("../../__internal__/utils/helpers/guid");
 const mockedGuid = "guid-12345";
-guid.mockImplementation(() => mockedGuid);
+(guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
+
+function renderTextarea(props?: TextareaProps, renderer = mount) {
+  return renderer(<Textarea name="textarea" {...props} />);
+}
 
 describe("Textarea", () => {
-  let wrapper;
+  let wrapper: ReactWrapper;
 
   testStyledSystemMargin((props) => <Textarea {...props} />);
   describe("when textarea is rendered with default props", () => {
-    let textarea;
+    let textarea: HTMLInputElement;
 
     beforeAll(() => {
       wrapper = renderTextarea();
-      textarea = wrapper.find("textarea").instance();
+      textarea = (wrapper
+        .find("textarea")
+        .instance() as unknown) as HTMLInputElement;
```

Comment:
we should not test on instances, I think that it is a good moment to rewrite this test with that refactor.

Reply:
agreed, I didn't even know what this meant, and having looked it up I'm still confused as to how it works with functional components, or (as here) plain HTML elements!

It looks like we want to get the DOM element, so `.getDOMNode()` instead of `.instance()` ought to work I would think.

Reply:
This is what I wanted to speak about. I spoke to @edleeks87 about this in the office, as I also wasn't sure what it meant. He suggested that the test was kind of superfluous and realistically this ticket wasn't to address these kinds of issues - it will be addressed when we move to RTL. However, we can remove the instance check and do `.getDOMNode()` as you suggest Robin and that works. We just wanted to check if this is what you meant here, @grabkowski?